### ISSUE TITLE
Rocky Linux not correctly handled in ntopng-utils-manage-updates

### ISSUE THREAD

**Issue by lumarel** [ntop/ntopng#6491]
**Environment**:
* OS name: Rocky Linux
* OS version: 8.5
* Architecture: x86_64
* ntopng version/revision: 5.3.220328-17343 (package from the EL8 repo)

**What happened**:

cron runs the ntopng-utils-manage-updates script every minute,
and produces a dpkg error every time it runs.

This happens due to line 57 not expecting "Rocky".

**What did you expect to happen?**

ntopng-utils-manage-updates should run without error.

**How did you reproduce it?**

- Wait until cron runs the command and look at the system journal
- Or execute `/usr/bin/ntopng-utils-manage-updates -a handle-on-demand-requests`

**Debug Information**:

```
Apr 03 17:33:01 r8-ntopng-prod CROND[255937]: (root) CMD (/usr/bin/ntopng-utils-manage-updates -a handle-on-demand-requests)
Apr 03 17:33:01 r8-ntopng-prod CROND[255936]: (root) CMDOUT (/usr/bin/ntopng-utils-manage-updates: line 69: dpkg: command not found)
```

Thank you for the great product! 🙂

**Comment by cardigliano** [comment]
Fixed in dce7bbeebfb8ca30e896c2b4f8b4e7867f067d5d , please check the next build

**Comment by lumarel** [comment]
I can confirm, this fixed the issue!
Thank you very much for the fast response and fix 🎉
